collections-lists/ConvertIAsyncEnumerableToList/ConvertIAsyncEnumerableToListLibrary/UserPageFetcher.cs

```diff
@@ -0,0 +1,49 @@
+﻿using ConvertIAsyncEnumerableToListLibrary.Model;
+using System.Runtime.CompilerServices;
+
+namespace ConvertIAsyncEnumerableToListLibrary;
+public static class UserPageFetcher
+{
+    public static async IAsyncEnumerable<User> GetUsersAsync(int delayMilliseconds = 10)
```

Comment:
(1 of 3) I think you can leave this as 500, in both methods.
```suggestion
    public static async IAsyncEnumerable<User> GetUsersAsync(int delayMilliseconds = 500)
```